src/backend/base/langflow/components/deactivated/ChatLiteLLMModel.py

```diff
@@ -168,13 +161,12 @@ def build_model(self) -> LanguageModel:
             n=self.n,
             max_tokens=self.max_tokens,
             max_retries=self.max_retries,
-            openai_api_key=api_keys["openai_api_key"],
-            azure_api_key=api_keys["azure_api_key"],
-            anthropic_api_key=api_keys["anthropic_api_key"],
-            replicate_api_key=api_keys["replicate_api_key"],
-            cohere_api_key=api_keys["cohere_api_key"],
-            openrouter_api_key=api_keys["openrouter_api_key"],
+            openai_api_key=self.api_key,
```

Comment:
Won't this pass the same variable to all? I assume only one api key must be passed, otherwise it will try to run the wrong one.